src/Viz.js

```diff
@@ -660,6 +672,16 @@ If *data* is not specified, this method returns the current primary data array,
     return arguments.length ? (this._depth = _, this) : this._depth;
   }
 
+  /**
+      @memberof Viz
+      @desc If *value* is specified, sets the description accessor to the specified function or string and returns the current class instance.
+      @param {Function|String} [*value*]
+      @chainable
+  */
+  desc(_) {
+    return arguments.length ? (this._desc = typeof _ === "function" ? _ : constant(_), this) : this._desc;
```

Comment:
`this._desc` should only ever be a *String*, so remove the function code from here